datashader/pipeline.py

```diff
@@ -59,14 +42,14 @@ class Pipeline(param.Parameterized):
     glyph = param.ClassSelector(glyphs.Glyph,default=glyphs.Point("x","y"), doc="""
         Marker shape for each point, specified using fields for the data object.""")
 
-    agg = param.ClassSelector(reductions.Reduction, default=reductions.count("count"), 
+    agg = param.ClassSelector(reductions.Reduction, default=reductions.count(), 
         doc="""Function for incrementally reducing a bin's values into a scalar.""")
 
     transfer_fns = param.HookList(default=[], doc="""
         Optional function(s) to apply to the aggregated bin values, before
         they each get converted into a color.""")
 
-    color_fn = param.Callable(default=Interpolate(), doc="""
+    color_fn = param.Callable(default=partial(tf.interpolate,low="lightblue",high="darkblue"), doc="""
```

Comment:
History -- I changed the default here first, and only then on tf.interpolate.  Sure, I should delete it here, for simplicity.